tests/test_data/detection/1/per_sweep_annotations_amodal/tracked_object_labels_2.json

```diff
@@ -0,0 +1,4 @@
+[
+  {"center": {"x": 0, "y": 0, "z": 0}, "rotation": {"x": 1.0, "y": 0.0, "z": 0, "w": 0}, "length": 2, "width": 2, "height": 2, "track_label_uuid": "00000000-0000-0000-0000-000000000000", "timestamp": 2, "label_class": "VEHICLE"},
```

Comment:
is `prettyprint` of the json possible? so that the line lengths are still 80 chars?